README.md

```diff
@@ -50,6 +50,7 @@ The `client` feature requires enabling one random number generator:
 
 - [`fastrand`](https://docs.rs/fastrand/latest/fastrand) can be used as a `PRNG`
 - [`getrandom`](https://docs.rs/getrandom/latest/getrandom) can be used as a cryptographically secure RNG
+- [`nightly`](https://doc.rust-lang.org/stable/std/random/struct.DefaultRandomSource.html) can be used as a cryptographically secure RNG
```

Comment:
Hmm, I think this would better be phrased something like this:

> `std::random` can be used as a cryptographically secure RNG with the `nightly` feature enabled

since nightly isn't the RNG, it's std::random